### PR TITLE
[13.0][IMP] partner_company_group: Add advises when company_group pricelist is different than member pricelists

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT35300

please @carlosdauden @sergio-teruel review this

### CODE REVIEW

File: partner_company_group/models/res_partner.py [hunk 1]
```diff
@@ -18,3 +18,64 @@ class Contact(models.Model):
...
+                "message": _(
+                    "The company group %s has the pricelist %s, that is different than"
+                    " the pricelist set on this contact"
+                    % (
+                        self.company_group_id.display_name,
+                        self.company_group_id.property_product_pricelist.display_name,
+                    )
+                ),
+            }
+        elif (
```
Comment (carlosdauden) [inline partner_company_group/models/res_partner.py]: You could extract the common code to a new method to not repeat the same code ([DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)).
Or in this case call direcly to _onchange_company_group_id
```suggestion
        res = self._onchange_company_group_id()
        if not res:
```

**Review by carlosdauden — APPROVED** [review]

**Comment by CarlosRoca13** [comment]
Changed!

**Review by sergio-teruel — CHANGES_REQUESTED** [review]
@CarlosRoca13 please add some test to coverage this behavior

**Comment by sergio-teruel** [comment]
ping @CarlosRoca13

**Comment by CarlosRoca13** [comment]
Sorry for the delay, added

**Review by sergio-teruel — APPROVED** [review]

**Review by rousseldenis — COMMENTED** [review]
LGTM

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**